### PR TITLE
Include thumbnail url if file set is public

### PR DESCRIPTION
**Pull request by cjcolvar**
The request tests are a not testing as much as I'd like because the file sets aren't getting processed by the test suite so a derivative file isn't being generated to satisfy the test in the thumbnail indexer: https://github.com/samvera/hyrax/blob/2.x-stable/app/services/hyrax/thumbnail_path_service.rb#L49

### CODE REVIEW

File: app/views/hyku/api/v1/collection/_collection.json.jbuilder [hunk 1]
```diff
@@ -14,6 +14,11 @@ json.resource_type collection.resource_type&.first
 json.rights_statements_for_api_tesim collection.rights_statement&.first
 json.thumbnail_base64_string nil
 json.thumbnail_url collection.thumbnail_path
+if Hyrax::PresenterFactory.build_for(ids: [collection.thumbnail_id], presenter_class: Hyrax::FileSetPresenter, presenter_args: [current_ability, request]).first&.solr_document&.public?
```
Comment (prdanelli) [inline app/views/hyku/api/v1/collection/_collection.json.jbuilder:17]: Could we establish a maximum line length? 120 is the norm for this and makes code much easier to read - IMO. This one is 184. 

It could be flagged as part of the rubocop configuration.

**Review by prdanelli — APPROVED** [review]

File: app/views/hyku/api/v1/collection/_collection.json.jbuilder (hunk 1 above)
Comment (cjcolvar) [reply app/views/hyku/api/v1/collection/_collection.json.jbuilder:17]: That's a good idea.  The Samvera community style standards have it bumped up to 200 (See https://github.com/samvera-labs/bixby/blob/master/bixby_default.yml#L470).  We could lower that here but I'd like to do that in a separate PR since it looks like it affects 84 lines.  Maybe we could also lobby for changing the community standard so we don't have to override it here.

Comment (prdanelli) [reply app/views/hyku/api/v1/collection/_collection.json.jbuilder:17]: OMG 200!? That seems incredibly long to me. 

I am a little dyslexic and longer lines are hard to parse for me. 

Id be happy to look at this PR so that we could reduce it down to a manageable level.

Comment (cjcolvar) [reply app/views/hyku/api/v1/collection/_collection.json.jbuilder:17]: I think you're experiences are a good reason to push for lowering the limit.  Do you mind writing up an issue for bixby?
https://github.com/samvera-labs/bixby/issues/new
